cloud/amazon/cloudtrail.py

```diff
@@ -165,9 +165,11 @@ def main():
     if not HAS_BOTO:
       module.fail_json(msg='boto is required.')
 
+    security_token = module.params['security_token']
```

Comment:
Should be updated to use get_aws_connection_info function. See https://github.com/ansible/ansible-modules-core/blob/devel/cloud/amazon/ec2.py for an example.